Documentation/exposing-prometheus-and-alertmanager.md

```diff
@@ -0,0 +1,291 @@
+# Exposing Prometheus and Alertmanager
+
+The Prometheus Operator takes care of operating Prometheus and Alertmanagers
+clusters, however, there are many ways in Kubernetes to expose these to the
+outside world. This document outlines best practices and caveats to do so in
+various ways.
+
+## NodePort
+
+The easiest way to expose Prometheus or Alertmanager is to use a `Service` of
+type `NodePort`.
+
+A simple Prometheus object's manifest could look like this:
+
+```
+apiVersion: "monitoring.coreos.com/v1alpha1"
+kind: "Prometheus"
+metadata:
+  name: "main"
+spec:
+  replicas: 1
+  version: v1.5.0
+  resources:
+    requests:
+      memory: 400Mi
+```
+
+All Prometheus `Pod`s are labelled with `prometheus: <prometheus-name>`, as the
+Prometheus object's name is `main`, the selector ends up being `prometheus:
+main`. Meaning, the respective manifest for the `Service` would look like this:
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: prometheus-main
+spec:
+  type: NodePort
+  ports:
+  - name: web
+    nodePort: 30900
+    port: 9090
+    protocol: TCP
+    targetPort: web
+  selector:
+    prometheus: main
+```
+
+After creating a `Service` with the above manifest, the web UI of Prometheus
+will be accessible by browsing to any of the worker nodes using
+`http://node-ip:30900/`.
```

Comment:
Should clearly indicate the interpolated part like `http://<node-ip>:30900/`

Equally below.